ethclient/ethclient.go

```diff
@@ -393,6 +393,14 @@ func (ec *Client) SubscribeFilterLogs(ctx context.Context, q ethereum.FilterQuer
 	return ec.c.EthSubscribe(ctx, ch, "logs", arg)
 }
 
+// ToFilterArg is for testing purposes only.
+func ToFilterArg(q ethereum.FilterQuery, test bool) (interface{}, error) {
```

Comment:
@fjl I had to blackbox `ethclient_test.go` to prevent a circular import, so is this method okay for testing `toFilterArg`?